cloudformation/media-atom-maker-dev.yml

```diff
@@ -115,9 +115,30 @@ Resources:
           ProvisionedThroughput:
             ReadCapacityUnits: "5"
             WriteCapacityUnits: "5"
+
+  UploadTrackingDynamoTable:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: "atomId"
+          AttributeType: "S"
+        - AttributeName: "id"
+          AttributeType: "S"
+      KeySchema:
+        - AttributeName: "atomId"
+          KeyType: "HASH"
+        - AttributeName: "id"
+          KeyType: "RANGE"
+      ProvisionedThroughput:
+        ReadCapacityUnits: "5"
+        WriteCapacityUnits: "5"
+      StreamSpecification:
+        StreamViewType: "NEW_IMAGE"
+
   UserUploadBucket:
     Type: "AWS::S3::Bucket"
     Properties:
+      BucketName: "media-atom-maker-upload-dev"
```

Comment:
A fixed bucket name is required to avoid a circular dependency with the lambda trigger